tests/Aql/TraversalExecutorTest.cpp

```diff
@@ -1,715 +0,0 @@
-////////////////////////////////////////////////////////////////////////////////
-/// DISCLAIMER
-///
-/// Copyright 2014-2020 ArangoDB GmbH, Cologne, Germany
-/// Copyright 2004-2014 triAGENS GmbH, Cologne, Germany
-///
-/// Licensed under the Apache License, Version 2.0 (the "License");
-/// you may not use this file except in compliance with the License.
-/// You may obtain a copy of the License at
-///
-///     http://www.apache.org/licenses/LICENSE-2.0
-///
-/// Unless required by applicable law or agreed to in writing, software
-/// distributed under the License is distributed on an "AS IS" BASIS,
-/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-/// See the License for the specific language governing permissions and
-/// limitations under the License.
-///
-/// Copyright holder is ArangoDB GmbH, Cologne, Germany
-///
-/// @author Michael Hackstein
-////////////////////////////////////////////////////////////////////////////////
-
-#include "gtest/gtest.h"
```

Comment:
Why are we removing this file?
As far as I see it should be testing the AQL ExecutionNode.
Should we not move this to use a MockGraph / MockGraphProvider?
It should also be able to use a SingleServerTraversal here.

Reply:
Discussed to delete it for now, and potentially revive it later on.
We kept it in Kansan history.
The items under test here are covered by Integration tests anyways.

Reply:
Created card